modules/ai/assets/js/editor/components/voice-promotion-alert.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Alert } from '@elementor/ui';
+import BulbIcon from '../icons/bulb-icon';
+import useIntroduction from '../hooks/use-introduction';
+
+export const VoicePromotionAlert = ( props ) => {
+	const { isViewed, markAsViewed } = useIntroduction( props.introductionKey );
+
+	return (
+		! isViewed
+			?			<Box sx={ { mt: 2, ...props.sx } } alignItems="top">
```

Comment:
It can be more readable

```suggestion

	if ( ! isViewed ) {
		return null;
	}

	return (
		<Box sx={ { mt: 2, ...props.sx } } alignItems="top">
```